backend/app/models/models.py

```
@@ -0,0 +1,490 @@
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)